src/YahooFinance.py

```diff
@@ -6,103 +6,103 @@
 
```

Comment:
From what I can tell, nothing changed in this file except for the `URL_TEMPLATE`. Can we revert all of the whitespace changes and just change line `12` if that's the only functionality change?